tcms/rpc/tests/test_testcase.py

```diff
@@ -430,6 +507,27 @@ def test_fails_when_mandatory_fields_not_specified(self):
             )
 
 
+class TestRemovePermissions(APIPermissionsTestCase):
+    permission_label = 'testcases.delete_testcase'
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.case_1 = TestCaseFactory()
+        self.case_2 = TestCaseFactory()
+        self.case_3 = TestCaseFactory()
+
+        self.query = {'pk__in': [self.case_1.pk, self.case_2.pk, self.case_3.pk]}
+
+    def verify_api_with_permission(self):
+        len_deleted, num_deleted = self.rpc_client.TestCase.remove(self.query)
```

Comment:
`num_deleted` is the first return value, the second one was unused, right ?

Reply:
I got a little confused and changed the name of first return value. Both of them are number of deletions with respect to different things. :sweat_smile: 

I have fixed it. :smile: